### PR TITLE
Add `VecNormalize` support

### LINKED ISSUE #6: Add support for VecNormalize
Detect and save automatically normalization when present (using for instance `unwrap_vec_normalize()` helper).

### PR DESCRIPTION
**Pull request by araffin**
- add missing type hints
- fix `push_to_hub` (bug detected by pytype checker)
- cleanup
- add support for `VecNormalize`

closes #6 

Demo (and training/loading code): https://huggingface.co/araffin/a2c-Pendulum-v1

### CODE REVIEW

File: huggingface_sb3/push_to_hub.py [hunk 1]
```diff
@@ -283,10 +306,19 @@ def package_to_hub(
...
     msg.info(
-        "This function will save, evaluate, generate a video of your agent, create a model card and push everything to the hub. It might take up to 1min. \n This is a work in progress: If you encounter a bug, please open an issue and use push_to_hub instead."
+        "This function will save, evaluate, generate a video of your agent, "
+        "create a model card and push everything to the hub. "
+        "It might take up to 1min. \n "
+        "This is a work in progress: if you encounter a bug, please open an issue "
+        "and use push_to_hub instead."
     )
-    huggingface_token = HfFolder.get_token()
+    HfFolder.get_token()
```
Comment (simoninithomas) [inline huggingface_sb3/push_to_hub.py]: - We can remove `HfFolder.get_token()`

**Review by simoninithomas — CHANGES_REQUESTED** [review]
Awesome work thank you very much Antonin, the code is cleaner and the vecnorm process is really smooth. Just added a small change with `HfFolder.get_token()` we can definitely remove it now with the new version of HuggingFace_hub 🤗 .


cc @osanseviero.

File: huggingface_sb3/push_to_hub.py (hunk 1 above)
Comment (araffin) [reply huggingface_sb3/push_to_hub.py]: Done in https://github.com/huggingface/huggingface_sb3/pull/10/commits/7788b5a1ab63b3eef4cd6a1e1da929b7bdd3a371

**Review by simoninithomas — APPROVED** [review]

**Comment by osanseviero** [comment]
We should push a new tag with latest commit to trigger a release @simoninithomas

**Pull request merged**